src/Constraints/SsnValidator.php

```diff
@@ -18,10 +18,14 @@ public function validate($value, Constraint $constraint)
     {
         parent::validate($value, $constraint);
 
-        // Have to instantiate it. See: https://github.com/ronanguilloux/IsoCodes/issues/12
-        $ssn = new IsoCodes\Ssn();
+        if (in_array('IsoCodes\IsoCodeInterface', class_implements('IsoCodes\Ssn'), true)) {
```

Comment:
Is there a reason not to use `if ($ssn instanceof IsoCodes\IsoCodeInterface) {` instead of this?